src/Android/Renderers/CustomPageRenderer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Android.App;
+using Android.Content;
+using AndroidX.AppCompat.Widget;
+using Bit.App.Resources;
+using Bit.Droid.Renderers;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.Android;
+
+[assembly: ExportRenderer(typeof(ContentPage), typeof(AndroidNavigationPageRenderer))]
+namespace Bit.Droid.Renderers
+{
+    public class AndroidNavigationPageRenderer : PageRenderer
+    {
+        public AndroidNavigationPageRenderer(Context context) : base(context)
+        {
+        }
+
+        protected override void OnElementChanged(ElementChangedEventArgs<Page> e)
+        {
+            base.OnElementChanged(e);
+
+            Activity context = (Activity)this.Context;
+            var toolbar = context.FindViewById<Toolbar>(Resource.Id.toolbar);
+            if(toolbar != null)
+            {
+                toolbar.NavigationContentDescription = AppResources.GoBack;
+            }
+        }
+    }
+}
```

Comment:
I have to check, but isn't there a way to set this with an `Effect` or a custom renderer for the `Toolbar` instead of the whole page?

Reply:
i'll take a look if it works with an effect or a `Toolbar` renderer.

Reply:
Toolbar support seems pretty weak. Didn't find a better solution to add the accessibility text to the toolbar.